doc: update README & add QR code (#2218)
apache/incubator-hugegraph: README.md

3. [hugegraph-commons](https://github.com/apache/incubator-hugegraph-commons) (**common & rpc** module)
4. [hugegraph-website](https://github.com/apache/incubator-hugegraph-doc) (**doc & website** code)

## Contact Us
- Github Issues: Feedback on usage issues and functional requirements (priority)

> - Feedback Email: dev@hugegraph.apache.org

Reviewer: Hi, @1216063060, you can refer this PR https://github.com/apache/incubator-hugegraph-toolchain/pull/462, pay attention to adding blank lines between title and content,  remove extra blank lines in contents.

Reviewer 2: OK，3Q~

3. [hugegraph-commons](https://github.com/apache/incubator-hugegraph-commons) (**common & rpc** module)
4. [hugegraph-website](https://github.com/apache/incubator-hugegraph-doc) (**doc & website** code)

> 

Reviewer 3: remove extra line

Thanks to TinkerPop, thanks to Titan, thanks to DataStax. Thanks to all other organizations or authors who contributed to the project.

You are welcome to contribute to HugeGraph, and we are looking forward to working with you to build an excellent open source community.

Contact Us

> ---

Reviewer 3: add a space after the separate line

> Contact Us
> ---

Reviewer 3: ```suggestion
### Contact Us

---

```

>  - [GitHub Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (priority)

Reviewer 3: usr `high priority` instead of `priority`

>  - [GitHub Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (priority)

Reviewer 3: ```suggestion
 - [GitHub Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (quick response)
```

>  - Feedback Email: [dev@hugegraph.apache.org](mailto:dev@hugegraph.apache.org)

Reviewer 3: ```suggestion
 - Feedback Email: [dev@hugegraph.apache.org](mailto:dev@hugegraph.apache.org) ([subscriber](https://hugegraph.apache.org/docs/contribution-guidelines/subscribe/) only)
```

notice people we need subscribe the dev mailing-list first

Thanks to TinkerPop, thanks to Titan, thanks to DataStax. Thanks to all other organizations or authors who contributed to the project.

You are welcome to contribute to HugeGraph, and we are looking forward to working with you to build an excellent open source community.

## Contact Us

- [Github Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (priority)
- Feedback Email: [dev@hugegraph.apache.org](dev@hugegraph.apache.org)
- WeChat public account: Apache HugeGraph, welcome to scan this QR code to follow us.

> <img src="https://github.com/apache/incubator-hugegraph-doc/blob/master/assets/images/wechat.png?raw=true" alt="QR png" width="350" />

Reviewer 3: I use html tag to reduce the size of the image, other repo could refer this

Author: Hi @imbajin ，Is this the picture you were expecting 
![image](https://github.com/apache/incubator-hugegraph/assets/87920097/9f6fa0fe-2d78-4a84-8470-93cb71d0de6a)

> > <img src="https://github.com/apache/incubator-hugegraph-doc/blob/master/assets/images/wechat.png?raw=true" width="350" />

Reviewer 3: why use the quota for this section？（a mistake？）

Thanks to TinkerPop, thanks to Titan, thanks to DataStax. Thanks to all other organizations or authors who contributed to the project.

You are welcome to contribute to HugeGraph, and we are looking forward to working with you to build an excellent open source community.

## Contact Us

- [Github Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (priority)
- Feedback Email: [dev@hugegraph.apache.org](dev@hugegraph.apache.org)
- WeChat public account: Apache HugeGraph, welcome to scan this QR code to follow us.

> ![new QR code](https://user-images.githubusercontent.com/87920097/242012297-306495b5-18be-4f58-9335-21ee3c201408.png)

Reviewer 3: just use `QR-Code` as the name, and remember update the image url later
